components/dashboard/src/data/projects/fetchers.ts

```diff
@@ -6,26 +6,26 @@
 
 import { PrebuildWithStatus, Project } from "@gitpod/gitpod-protocol";
 import dayjs from "dayjs";
+import { useCallback } from "react";
 import { useFeatureFlags } from "../../contexts/FeatureFlagContext";
 import { listAllProjects } from "../../service/public-api";
 import { getGitpodService } from "../../service/service";
 
-type UseFetchProjectsArgs = {
+type UseProjectsFetcherArgs = {
```

Comment:
Changed some of the naming conventions here to be more consistent with why I ended up doing in workspaces